editor/src/components/canvas/gap-utils.ts

```diff
@@ -82,13 +83,34 @@ function paddingControlContainerBoundsFromChildBounds(
   }))
 }
 
+function inset(sidess: Sides, rect: CanvasRectangle): CanvasRectangle {
```

Comment:
not good because `sides` is the constructor function for `Sides`, and applying your suggestion would result in a lint error, since it happens to be in scope (I also named the param `sides` when I created the function)